Unable to start locally the documentation following the website/README.md file

## Description

The current instructions (website/README.md file) to run locally the website is not working on my environment, and some requirements are missed to do it.

I am using a Fedora 26 SO.

### Steps to reproduce

Add the steps required to reproduce this error.

1. Run `yarn` command. OK
2. Run `yarn copyDocs`. KO

```shell
❯ yarn copyDocs
yarn run v1.22.17
error Command "copyDocs" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

3. Run `npx docusaurus start` to start the server as it is described in the [Docusaurus documentation](https://docusaurus.io/docs)

```shell
❯ npx docusaurus start
[ERROR] Minimum Node.js version not met :(
[INFO] You are using Node.js v14.18.1, Requirement: Node.js >=16.14.
```

4. Using required Node.js version: `nvm use v16.14.2'
5. Start server with `npx docusaurus start`

```shell
❯ npx docusaurus start
[INFO] Starting the development server...
[SUCCESS] Docusaurus website is running at: http://localhost:3000/
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating

✔ Client
  Compiled successfully in 9.17s

Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
client (webpack 5.73.0) compiled successfully
```

### Expected vs actual behaviour

A better description of the steps to execute locally the website.

### Logs

N/A

--- a/website/README.md
+++ b/website/README.md
@@ -11,7 +11,7 @@ yarn
 ### Local Development
 
 ```shell
-npx docusaurus start
+yarn start
 ```
 
 This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.

--- a/website/README.md
+++ b/website/README.md
@@ -11,10 +11,12 @@ yarn
 ### Local Development
 
 ```shell
-yarn copyDocs
+npx docusaurus start
 ```
 
 This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+**NOTE:** Requires Node.js >=16.14.
 
 ### Build
 

